includes/class-wc-gateway-abstract-maksuturva.php

```diff
@@ -319,7 +319,7 @@ abstract class WC_Gateway_Abstract_Maksuturva {
 		'pmt_errorreturn',          // Alphanumeric  max-length: 200  -.
 		'pmt_cancelreturn',         // Alphanumeric  max-length: 200  -.
 		'pmt_delayedpayreturn',     // Alphanumeric  max-length: 200  -.
-		'pmt_escrow',               // Alpha         max-length: 1    min-length: 1   Maksuturva=Y, eMaksut=N.
+		'pmt_escrow',               // Alpha         max-length: 1    min-length: 1   Svea=Y, eMaksut=N.
```

Comment:
Everything after min-length: 1 can be removed altogether. That is, Y/N description can be removed.